### PR TITLE
Include CSIDriver creation in Operator

### PR DESCRIPTION
**Pull request by huffmanca**
Allows the operator to create the Azure Disk CSIDriver on the server.

### CODE REVIEW

File: assets/csidriver.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  name: disk.csi.azure.com
+  annotations:
+      # This CSIDriver is managed by an OCP CSI operator
+      csi.openshift.io/managed: "true"
```
Comment (bertinatto) [inline assets/csidriver.yaml:7]: Who reads this annotation?

I see AWS, oVirt and Manila have this annotation, but others don't. I think it's a leftover from the OLM version of AWS operator.

Comment (huffmanca) [reply assets/csidriver.yaml]: I've removed the annotation for now; however, couldn't we use this to determine if the customer has a 3rd-party CSI driver already installed?

Comment (bertinatto) [reply assets/csidriver.yaml]: I think we could, yes; however, we need to settle on the annotation name first, maybe there's a better name.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**